server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/mail/EmailSet.scala

```diff
@@ -207,6 +310,7 @@ case class ValidatedEmailSetUpdate private (keywordsTransformation: Function[Key
 class EmailUpdateValidationException() extends IllegalArgumentException
 case class InvalidEmailPropertyException(property: String, cause: String) extends EmailUpdateValidationException
 case class InvalidEmailUpdateException(property: String, cause: String) extends EmailUpdateValidationException
+case class AttachmentNotFoundException(throwable: Throwable, cause: String) extends EmailUpdateValidationException
```

Comment:
There is already a AttachmentNotFoundException in mailbox-api. I find this new one confusing...